views/default/form_Element_OphInBiometry_Selection_fields.php

```diff
@@ -2,7 +2,15 @@
 <div class="element-fields">
 	<div class="row">
 		<div class="large-12 column">
-			<?php echo $form->dropDownList($element, 'lens_id_'.$side, CHtml::listData(OphInBiometry_LensType_Lens::model()->activeOrPk($element->{'lens_id_'.$side})->findAll(array('order'=> 'display_order asc')),'id','name'),array('empty'=>'- Please select -'),null,array('label'=>3, 'field'=>6))?>
+			<?php
+			$post = Yii::app()->request->getPost('Element_OphInBiometry_Selection');
```

Comment:
1) No controller defined for this - we keep it in the view for now
2) That is the requirement to be able to save data without selecting a lens option